src/millisecondClock.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <chrono>
+
+class MillisecondClock {
```

Comment:
On the note of naming, note that you actually violated your own `camelCase` rule in this case. I named the file the way you would have named it otherwise, but I didn't really modify any code so I left the class name alone. You can correct that in a future change if you like.